services/httpd/handler.go

```diff
@@ -1796,7 +1796,7 @@ func (h *Handler) recovery(inner http.Handler, name string) http.Handler {
 
 // Store describes the behaviour of the storage packages Store type.
 type Store interface {
-	Read(ctx context.Context, req *datatypes.ReadRequest) (reads.ResultSet, error)
+	ReadFilter(ctx context.Context, req *datatypes.ReadFilterRequest) (reads.ResultSet, error)
```

Comment:
Is this a piece of another PR?

Reply:
Yea. This was a change to the storage interface.